src/agent/testrunpublisher.ts

```diff
@@ -4,6 +4,7 @@ import ctxm = require('./context');
 import cm = require('./common');
```

Comment:
The tests we have today are very basic. If we re going to make the js implementation as default one, we should definitely add more tests. Even if not as a part of this story, may be we take this up as a separate work item.